tests/KdybyTests/Console/ContainerHelper.phpt

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Test: Kdyby\Console\ContainerHelper.
+ *
+ * @testCase KdybyTests\Console\ContainerHelperTest
+ * @author Martin Procházka <juniwalk@outlook.cz>
+ * @package Kdyby\Console
+ */
+
+namespace KdybyTests\Console;
+
+use Kdyby\Console\ContainerHelper;
+use Nette\DI\Container;
+use Tester;
+use Tester\Assert;
+use Tracy\Debugger;
+
+
+require_once __DIR__ . '/../bootstrap.php';
+
+/**
+ * @author Martin Procházka <juniwalk@outlook.cz>
+ */
+class ContainerHelperTest extends \Tester\TestCase
+{
+	/**
+	 * @return Container
+	 */
+	private function createContainer()
+	{
+		$container = new Container([
+			'foo' => 'bar',
+		]);
+
+		return $container;
+	}
+
+
+	public function testContainer()
+	{
+		Debugger::$logDirectory = TEMP_DIR . '/log';
+		Tester\Helpers::purge(Debugger::$logDirectory);
+
+		$container = $this->createContainer();
+		$helper = new ContainerHelper($container);
+
+		Assert::type($container, $helper->getContainer());
+		Assert::same('container', $helper->getName());
+	}
+
+
+	public function testParameters()
+	{
+		Debugger::$logDirectory = TEMP_DIR . '/log';
+		Tester\Helpers::purge(Debugger::$logDirectory);
```

Comment:
Beats me, I had some weird errors before I rewrote that test, I will try to remove them.